mgmt/Cargo.toml

```diff
@@ -16,4 +16,6 @@ tokio = { workspace = true }
 tonic = { workspace = true }
 tracing = { workspace = true }
 tracing-test = { workspace = true }
+socket2 = { workspace = true }
```

Comment:
This one is not in the workspace's Cargo.toml

Reply:
yes, leftovers of my previous attempt to adjust socket buff